examples/certs/run.bash

```
@@ -0,0 +1,221 @@
+        echo '      - name: kubeconfig-consumer'
+        echo '        secret:'
+        echo '          secretName: kubeconfig-consumer'
+        echo '      - name: kubeconfig-internalca'
+        echo '        secret:'
+        echo '          secretName: kubeconfig-internalca'
+        echo '      - name: kubeconfig-externalca'
+        echo '        secret:'
+        echo '          secretName: kubeconfig-externalca'
+    } | kubectl::apply "$kind_platform" "-"
+    KUBECONFIG="$kubeconfigs/platform.kubeconfig" \
+        kubectl rollout status deployment/resource-broker -n default --timeout="$timeout" \
```

Shellcheck (0.11.0)

[warning] timeout is referenced but not assigned.

(SC2154)